go-packages/meep-loc-serv-notification-client/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-loc-serv-notification-client
+module github.com/InterDigitalInc/AdvantEDGE/go-packages/meep-rnis-notification-client
```

Comment:
Why is this RNIS?

Reply:
content of rnis and loc-serv file were reversed.....?..?

fixed